pkgs/games/mindustry/default.nix

```diff
@@ -8,6 +8,9 @@
 , gradle
 , jdk17
 
+# from https://github.com/NixOS/nixpkgs/issues/332831
```

Comment:
Usually comments like this one go near the use site. In this case I'd remove it entirely, since it's just a regular runtime exe dependency

```suggestion
```

Reply:
'k, i'll commit and rebase.